pom.xml

```diff
@@ -30,7 +30,7 @@
     <version.uuid-generator>3.2.0</version.uuid-generator>
     <version.camunda.commons>1.10.0</version.camunda.commons>
     <version.camunda.connect>1.5.2</version.camunda.connect>
-    <version.camunda.spin>1.12.0</version.camunda.spin>
+    <version.camunda.spin>1.13.0-SNAPSHOT</version.camunda.spin>
```

Comment:
I think we should rather release `1.13.0` immediately and use it here to prevent delays in the next platform alpha release.